kobo/apps/project_ownership/tasks.py

```diff
@@ -264,16 +264,42 @@ def task_restarter():
     stuck_threshold = timezone.now() - timedelta(
         minutes=config.PROJECT_OWNERSHIP_STUCK_THRESHOLD
     )
+    restarted_transfers = []
+    for transfer_status in TransferStatus.objects.filter(
```

Comment:
I thought we said we would start the "pending" after the stuck tasks? 
Is still the first thing to happen because of `MAX_RESTARTED_TRANSFERS` limit?